lib/eco/export/jruby.py

```diff
@@ -0,0 +1,344 @@
+# import copy
+import os
+import os.path
+import tempfile
+import subprocess
+
+from incparser.annotation import Annotation, ToolTip, Railroad
+
+from incparser.astree import EOS
+from grammar_parser.gparser import MagicTerminal, IndentationTerminal
+
+from PyQt4.QtCore import QSettings
+
+
+class JRubyMorphismLine(Annotation):
+
```

Comment:
Some classes have blank lines after their defs; some don't. I don't mind which we use, provided it's at least consistent within the file and, preferably, consistent with the rest of Eco.